app/javascript/application.js

```diff
@@ -1,3 +1,6 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
-import "@hotwired/turbo-rails"
-import "controllers"
+import '@hotwired/turbo-rails'
```

Comment:
Apologies for the single-quoting. Not sure what happened there. Double quotes all the way!

Reply:
I am actually a single quote person :/ should I change?!?!

Reply:
Ah, thank you. That's good to know. Interesting...